### ISSUE TITLE
Additional message needed on clone

### ISSUE THREAD

**Issue by ericwinger** [dalehenrich/Rowan#54]
If you try to clone an already cloned repository, Rowan will attach to the existing clone. 

We need an additional message saying that even though the clone was not cloned, it still was attached.

**Comment by dalehenrich** [comment]
Here's the new message, please word smith:
```
The project is registered with the system, however, A clone for 'RowanSample1' already exists in '$GS_HOME/shared/repos//RowanSample1', so the clone operation is being skipped and the project will reference the existing location.
```

**Comment by ericwinger** [comment]
Better? Worse?
```
A clone for 'RowanSample1' already exists in '$GS_HOME/shared/repos//RowanSample1' so the clone operation is being skipped. 

The project will be registered at the existing location with Rowan.
```

**Comment by dalehenrich** [comment]
At the time the inform is signalled, the project has already been registered  at the existing location, so perhaps:
```
A clone for 'RowanSample1' already exists in '$GS_HOME/shared/repos//RowanSample1' so the clone operation is being skipped. 

The project has been registered at the existing location with Rowan.
```

**Comment by dalehenrich** [comment]
fixed with PR #112
